plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/data/transformers/EpochTimeAttributeTransformer.java

```diff
@@ -60,15 +61,16 @@ DateTimeFormatter getFormatter() {
             }
 
             @Override
-            long toRawValue(Instant instant) {
+            Long toRawValue(Instant instant) {
```

Comment:
If I'm not mistaken, `long` type from the old version gets boxed to be returned as `Object` anyways, so I hope this is acceptable